### ISSUE TITLE
taiga-front fails to install on Ubuntu Server 16.04.6

### ISSUE THREAD

**Issue by Gim6626** [taigaio/taiga-front#1760]
Trying to install development Taiga setup following https://taigaio.github.io/taiga-doc/dist/setup-development.html guide and faced troubles with frontend.

`$ gem install --user-install sass scss_lint`
leads to:
```
...
ERROR:  Error installing scss_lint:
        scss_lint requires Ruby version >= 2.4.
...
```
while most recent Ruby on Ubuntu 16.04 is 2.3.1-2~16.04.12.

Tried to install older version of scss_lint:
`gem install --user-install -v 0.38 scss_lint`

It worked, but more troubles ahead.

Trying:
`$ npm install`

Got:
```
...
../src/sass_context_wrapper.h:8:26: fatal error: sass/context.h: No such file or directory
...
```

Tried to fix it using:
`$ sudo apt install libsass-dev libsass0`

Worked, but it is not the end.

Finally, gulp refuses to work:
```
$ gulp
gulp: symbol lookup error: /home/taiga/taiga-front/node_modules/node-sass/vendor/linux-x64-57/binding.node: undefined symbol: libsass_version
```

Seems that libsass from Ubuntu repo is too old and does not support this symbol.

Don't know what to do with it. Could anybody help me? I think documentation should be updated to help people handle cases which I faced.

P.S.: By the way, command
`sudo apt-get install -y nodejs npm`
from manual leads to error:
`nodejs : Conflicts: npm`
and seems that it should be fixed to `sudo apt-get install -y nodejs`.

**Comment by alexhermida** [comment]
@Gim6626 yes, depending on your Ubuntu version the most recent version changes. The included version in Ubuntu 16.04 is 2.3, and you must install a newer version of Ruby. Check https://gorails.com/setup/ubuntu/16.04.
Regarding `npm` as long as the newest versions of node include the package, you don't need to include it. We will update the doc. Regardless, try to use a node versión between 7.7.2 and 10, I think that in new versions doesn't work. I recommend you to use some utility to manage different node environments like https://github.com/nodenv/nodenv or https://github.com/nvm-sh/nvm.

**Comment by Gim6626** [comment]
Thanks, but this does not help solve last problem with libsass.
Though I've solved it adding `LIBSASS_EXT="no"` in front of `npm install`.
Seems to work, don't know if it will badly affect some Taiga functions. Hope no.
